### PR TITLE
center labels on navtabs, use avatar instead of gear when possible

### PR DESCRIPTION
**Pull request by tscholl2**
fixes for #2836 and also center the labels on the navtabs on the top

### CODE REVIEW

**Comment by DrXyzzy** [comment]
I'm testing in cc-in-cc.

File: src/smc-webapp/app_shared.cjsx [hunk 1]
```diff
@@ -65,7 +65,7 @@ exports.NavTab = rclass
     propTypes :
         label           : rtypes.string
         label_class     : rtypes.string
-        icon            : rtypes.string
+        icon            : rtypes.oneOfType([rtypes.string,rtypes.object])
```
Comment (johnjeng) [inline src/smc-webapp/app_shared.cjsx]: Shouldn't this be `rtypes.element`?

Comment (tscholl2) [reply src/smc-webapp/app_shared.cjsx]: Probably. I need to look at the list of `rtypes` to get an idea of which ones there are. I can change that.

**Comment by DrXyzzy** [comment]
First project tab doesn't look right.

button-ctr

**Pull request merged**